instrumentation/hibernate/hibernate-4.0/src/main/java/io/opentelemetry/javaagent/instrumentation/hibernate/v4_0/TransactionInstrumentation.java

```diff
@@ -28,7 +28,7 @@
 final class TransactionInstrumentation implements TypeInstrumentation {
 
   @Override
-  public ElementMatcher<ClassLoader> classLoaderMatcher() {
+  public ElementMatcher<ClassLoader> classLoaderOptimization() {
     // Optimization for expensive typeMatcher.
```

Comment:
Probably all these comments can be replaced by more verbose javadoc of the base method?

Reply:
Honestly I completely forgot about these comments. I'll remove them.